### DISCUSSION TITLE
Show error when there is no connection with the data source

### DISCUSSION THREAD

**Discussion by Dionnelektra** [appbaseio/reactivesearch#1688] [Q&A]
I am working with my react project with reactivesearch.  My data source is from my local elasticsearch. Everything is working fine but I want to inform users when there is connection error. Example the index stated does not exist it returns _http://localhost:9200/sample/_msearch?_  How can I render an error that can be shown to users.

**Comment by siddharthlatest** [comment]
@Dionnelektra You can use the `renderError` prop of ReactiveList to do this. See [here](https://docs.appbase.io/docs/reactivesearch/v3/result/reactivelist/#props).

![](https://i.imgur.com/OC6sDuF.png)

**Reply by Dionnelektra** [reply]
can i use it in reactivebase?
